fauna/README.md

```diff
@@ -0,0 +1,72 @@
+# Agent Check: Fauna
+
+## Overview
+
+Fauna is a fully managed, distributed document-relational database. With this integration, you can stream your Fauna database query logs into Datadog. This enables you to view your queries in real time, and you can see the following information about your queries:
+1. Fauna Cost Measures
+   1. Read Ops
+   2. Write Ops
+   3. Compute Ops
+2. Query Time
+3. Query Size Measures
+   1. Request and response size
+4. Query shape
+   1. The full request body with Personal Identifiable Data removed.
+5. Query response code
+
+For the full list of fields available in the logs, see the [Fauna Query Log Record Format][1].
+
+## Setup
+
+### Installation
+
+1. Click the 'Connect Accounts' button on the [Fauna integration tile][3] to start the OAuth flow and connect your Fauna and Datadog accounts.
+2. Once directed to Fauna, login with the account you'd like to connect. You can skip this step if you're already logged into your Fauna account.
+3. You are redirected to the Fauna Create Integration page, where you can select the region group(s) or databases that you would like to stream logs into Datadog for. Then, click **Create**.
+4. When directed to Datadog, click **Authorize** to authorize Fauna to create an API key for your account that will be used to send the database query logs.
+
+Once this flow is complete, you are redirected to the [Fauna Integrations Page][2], which shows your active integration.
+
+Within the next ten minutes, the [Fauna Overview Dashboard][7] will begin showing metrics for any queries issued for the configured region group(s) or databases.
```

Comment:
Can you clarify that the metrics are logs? This could confuse users who are expecting metrics but only see logs. If you are planning to send in metrics, you will need to include a list of metrics in a [metadata](https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file) file within the pr.

Reply:
updated! apologies for the confusion